pages/store/[id].vue

```diff
@@ -157,7 +157,7 @@
                       v-else
                       class="w-[20px] h-[20px] my-0.5 bg-white rounded-full border border-gray-300"
                     />
-                    <div class="grow">
+                    <div class="flex-1">
```

Comment:
I think we can keep `grow`

Reply:
![截圖 2025-06-07 13 50 34](https://github.com/user-attachments/assets/3b6ac38e-0e12-46d5-9b6b-c184090d1792)

If we use `grow`, it would cause the unselected circle to get squished.
Or do you want to keep `grow` but fix the width of the checkbox?

Reply:
should add `shrink-0` to the unselected checkbox